### PR TITLE
[wip] modify clubb energy fixer

### PR DESCRIPTION
**Pull request by oksanaguba**
The fix is needed due to the bug introduced in the PR #3281. however, the fix itself cannot remove energy leaks from clubb due to the limiter in physics_update. the fix was tested wrt climb in an F case but there may be no urge to take it into master.

see also issue https://github.com/E3SM-Project/E3SM/issues/4449 .

[non-BFB] for everything with EAM and CLUBB.

### CODE REVIEW

**Comment by oksanaguba** [comment]
Climo results for the commit 0cf2457 are here https://web.lcrc.anl.gov/public/e3sm/diagnostic_output/onguba/theta/theta-clubb-leak.471017.0002-0006/def/ .

**Comment by oksanaguba** [comment]
energy balance for the current commit:

Screen Shot 2021-08-31 at 2 29 12 PM

**Comment by oksanaguba** [comment]
@wlin7 or @golaz  -- would you comment on this? either I can do more runs, or the issue (a very small leak of energy from clubb) remains and I will close this PR.

@vlarson -- after running with the change in this branch and with energy diagnostics, I do not see energy leak messages from clubb anymore. letting you know in case you want to look at this code.

**Comment by rljacob** [comment]
Its too late to add this for v2 WC but @kvcalvin might want it for 2.1

File: components/eam/src/physics/cam/clubb_intr.F90 [hunk 1]
```diff
@@ -2489,7 +2504,9 @@ subroutine clubb_tend_cam( &
          ptend_loc%v(i,k)   = (vm(i,k)-state1%v(i,k))*invrs_hdtime                  ! north-south wind
          ptend_loc%q(i,k,ixq) = (rtm(i,k)-rcm(i,k)-state1%q(i,k,ixq))*invrs_hdtime  ! water vapor
          ptend_loc%q(i,k,ixcldliq) = (rcm(i,k)-state1%q(i,k,ixcldliq))*invrs_hdtime ! Tendency of liquid water
-         ptend_loc%s(i,k)   = (clubb_s(k)-state1%s(i,k))*invrs_hdtime               ! Tendency of static energy
+
+         ptend_loc%s(i,k)   = cpair*( temp_a(i,k) - state1%T(i,k) )*invrs_hdtime    ! Tendency from clubb
+         ptend_loc%s(i,k)   = ptend_loc%s(i,k)    - se_dis_k(k)*invrs_hdtime        ! Tendency to conserve energy
```
Comment (oksanaguba) [inline components/eam/src/physics/cam/clubb_intr.F90:2509]: I need to replace += op here with just =

**Comment by rljacob** [comment]
This would have to be approved by BGC.  @kvcalvin

**Comment by rljacob** [comment]
@oksanaguba can you create an issue describing the bug?

**Comment by rljacob** [comment]
I'm looking for an issue that specifically mentions what was broken by #3281 and this PR will fix.

**Pull request closed without merging**